builder/openstack/step_create_volume.go

```diff
@@ -119,7 +124,7 @@ func (s *StepCreateVolume) Cleanup(state multistep.StateBag) {
 		return
 	}
 
-	if status != "available" {
+	if status != "available" && status != "error" {
```

Comment:
I wonder, is there a reason why we look at the status in this cleanup step? If the `s.volumeID` is set, we can attempt to cleanup the volume, regardless of its status, if it was available/error, it will have been created and will be cleaned-up, otherwise we can report that `Delete` failed for some reason and that manual cleanup is necessary (potentially).